packages/datadog_flutter_plugin/android/ndk_crash_reports_v2/network_information

```diff
@@ -0,0 +1 @@
+{"ev_size":40}{"connectivity":"network_not_connected"}
```

Comment:
how this file is used? it seems it shouldn't be here?

Reply:
Ugh, it shouldn't. Thanks for catching it.